### PR TITLE
Restore Package Sanity

### PR DESCRIPTION
**Pull request by angerman**
This is _DRAFT_.

A second part that "reverts" 9d729d8573a23bb0358f56406c488b81dc576311 and properly add ledger/1.1.0, as well as some proper network release are still missing.

### CODE REVIEW

**Comment by angerman** [comment]
This is _DRAFT_.

A second part that "reverts" 9d729d8573a23bb0358f56406c488b81dc576311 and properly add ledger/1.1.0, as well as some proper network release are still missing.

File: bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs [hunk 1]
```diff
@@ -26,7 +26,7 @@ genesisFunds :: forall era. IsShelleyBasedEra era
   => NetworkId -> ShelleyGenesis StandardShelley -> [(AddressInEra era, Lovelace)]
 genesisFunds networkId g
  = map (castAddr *** fromShelleyLovelace)
-     $ ListMap.toList
+     $ Map.toList
```
Comment (Jimbo4350) [inline bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs:29]: This removes an optimization. Do you intend to merge this PR into master?

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: cabal.project [hunk 2]
```diff
@@ -124,62 +125,21 @@ allow-newer:
 
 source-repository-package
   type: git
-  location: https://github.com/denisshevchenko/threepenny-gui
-  tag: 4ec92ded05ccf59ba4a874be4b404ac1b6d666b6
-  --sha256: 00fvvaf4ir4hskq4a6gggbh2wmdvy8j8kn6s4m1p1vlh8m8mq514
+  location: https://github.com/input-output-hk/moo
```
[the remark this answers is not in the document]
Comment (angerman) [reply cabal.project:128]: Cool, so we can just drop this! :D

File: bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs (hunk 1 above)
Comment (angerman) [reply bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs:29]: @Jimbo4350 it does. It's basically going back to ledger/1.0.0. 

My tentative plan is to next bump ledger to the 1.1.0 release, (and have a proper CHaP release) for it.
Then bump network to some release, though the situation there looks much less clear to me.

And then revert 9d729d8573a23bb0358f56406c488b81dc576311, at which point this is hopefully much closer to where we are _today_ with master.

If we can re-arrange the imports in a way to get ListMap from ledger/1.0.0, that would help reduce the carnage introduced in 9d729d8573a23bb0358f56406c488b81dc576311

File: cabal.project [hunk 3]
```diff
@@ -124,62 +125,21 @@ allow-newer:
...
-    network-mux
-    ouroboros-consensus
-    ouroboros-consensus-byron
-    ouroboros-consensus-cardano
-    ouroboros-consensus-protocol
-    ouroboros-consensus-shelley
-    ouroboros-network
-    ouroboros-network-framework
-    ouroboros-network-testing
+  location: https://github.com/vshabanov/ekg-json
```
[the remark this answers is not in the document]
Comment (angerman) [reply cabal.project:136]: So we can drop them in https://github.com/input-output-hk/cardano-node/blob/release/1.35/cabal.project as well?

File: bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs (hunk 1 above)
Comment (angerman) [reply bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs:29]: That being said, if we can come to an agreement that we revert to ledger/1.0.0 and the network as per 1.35.4 for now in master. And then have a second PR to try and bring in new _released_ dependencies of ledger and network, that would _certainy_ help. But it does mean we'll go back in time a bit.

**Review by deepfire — APPROVED** [review]

**Comment by angerman** [comment]
@deepfire please do _not_ try to merge this PR.

**Comment by Soupstraw** [comment]
I copied the `cabal.project` file from this PR to #4608 and made everything compile. I'll close this in favor of my PR.

**Pull request closed without merging**